config/routes.rb

```diff
@@ -61,6 +61,11 @@
       resources :dashboard, only: [:index]
       resources :likes, only: [:index, :create, :destroy]
       resources :adopter_applications, path: "applications", only: %i[index create update]
+      resources :adopted_pets, only: [:index] do
```

Comment:
https://github.com/rubyforgood/pet-rescue/pull/910/files#r1720837255 to follow up on this comment.

I think something like below is cleaner. 
```
resources :adopted_pets, only: [:index] do
  resources :files, only: [:index], module: :adopted_pets
end
```